api/v3/examples/Contribution/Create.php

```diff
@@ -55,11 +55,11 @@ function contribution_create_expectedresult() {
         'contact_id' => '1',
         'financial_type_id' => '1',
         'contribution_page_id' => '1',
-        'payment_instrument_id' => '4',
```

Comment:
This is wierd - I presume this is test-generated but it seems like an unexpected result